autogen.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+checkInstalled () {
+	which "$1" || { echo "$1 not installed"; exit 0; }
```

Comment:
`which` is not POSIX.
IIRC `type` is, but what i'd do here instead is simply trying to run the command with --help
i.e.
`$1 --help >/dev/null 2>&1 || ...`

Reply:
also the exit returnvalue needs to be 1

Reply:
> also the exit returnvalue needs to be 1

:+1: :D